src/utils/misc.jl

```diff
@@ -80,4 +80,12 @@ function doi2bib(doi::String)
         @warn "no internet connection"
         return ""
     end
+end
+
+function evalexppoly(x,n,v)
+    res = zero(x*first(n)*first(v))
+    for i in 1:length(n)
+        res += n[i]*x^v[i]
```

Comment:
I made a modification here since the previous implementation was giving an error

Reply:
Yeah, I saw, my mistake 😅